contrib/pom.xml

```diff
@@ -623,5 +628,11 @@
       <artifactId>super-csv-joda</artifactId>
       <version>2.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.gemstone.gemfire</groupId>
```

Comment:
Would you guys like to point it to Geode instead of GemFire ?  For now it would need to be pulled from incubator repository and from nightlies but a release is just about to start. 

```<dependency>
      <groupId>org.apache.geode</groupId>
      <artifactId>gemfire-core</artifactId>
      <version>${geode.version}</version>
</dependency>
```